docs/docs/developers/guides/js_apps/pay_fees.md

```diff
@@ -0,0 +1,96 @@
+---
+title: How to Pay Fees
+tags: [fees, transactions, developers]
+---
+
+There a several ways to pay for transaction fees, thanks to fee abstraction implemented in the Aztec protocol.
+
+This guide shows the various ways this can be done. For more of a background see [Fees on Aztec](../../../aztec/concepts/fees).
+
+## Methods to pay a tx fee
+
+### Natively via an existing balance of fee juice
+
+Either your own account or someone else's address has a balance of fee juice, and the transaction is paid for from this balance.
+
+### Natively via a balance of fee juice to be claimed
+
+Processing this transaction first claims bridged fee juice then is paid for from the balance, all in the same public/private tx.
+
+### Privately or publicly via a Fee Paying Contract
+
+A fee paying contract (FPC) is created and nominates a token that it accepts to then pay for txs in fee juice. So a user doesn't need to hold fee juice, they only need the token that the FPC accepts.
```

Comment:
would mention they can optionally claim refund too
in private - getting refund for excess repayment is very very powerful (sequencer refunds you money without knowing who you are! but this should be optional as it is costly in DA and proving time and for small txs may not make sense!)